src/coprocessor/dag/rpn_expr/types/function.rs

```diff
@@ -17,26 +17,18 @@
 //! itself. Instead, it creates a `foo_fn_meta()` function (simply add `_fn_meta` to the original
 //! function name) that generates an `RpnFnMeta` struct.
 //!
-//! If you needs `EvalContext` or the raw `RpnFnCallPayload`, just put it ahead of the function
-//! parameters, and add `ctx` or `payload` argument to the attribute. For example:
+//! In case of needing other parameters:
 //!
 //! ```ignore
 //! // This generates `with_context_fn_meta() -> RpnFnMeta`
-//! #[rpn_fn(ctx)]
+//! #[rpn_fn(capture = [ctx])]
 //! fn with_context(ctx: &mut EvalContext, param: &Option<Decimal>) -> Result<Option<Int>> {
 //!     // Your RPN function logic
 //! }
-//!
-//! // This generates `with_ctx_and_payload_fn_meta() -> RpnFnMeta`
-//! #[rpn_fn(ctx, payload)]
-//! fn with_ctx_and_payload(
-//!     ctx: &mut EvalContext,
-//!     payload: &RpnFnCallPayload<'_>
-//! ) -> Result<Option<Int>> {
-//!     // Your RPN function logic
-//! }
 //! ```
 //!
+//! All available parameters are: `ctx`, `output_rows`, `args`, `extra`.
```

Comment:
I think it may be clearer to put this together with L20. Now the "other parameters" in L20 looks confusing.